lib/logic/session-store.js

```diff
@@ -19,6 +19,24 @@ SessionStore.prototype.getRequestId = function() {
     return this.store.getItem(ID_REQUEST_ID);
 };
 
+SessionStore.prototype.getGithubApiEndpoint = function() {
+    return this.store.getItem("github_api_endpoint") || "https://api.github.com";
+};
+
+SessionStore.prototype.getSupplementaryApiEndpoint = function() {
+    return this.store.getItem("supplementary_api_endpoint") || "https://review.rocks";
+};
+
+SessionStore.prototype.getGithubDelays = function() {
```

Comment:
I wonder how much this should be persisted vs being constants in code. Justification for persisting this?